### ISSUE TITLE
Copyright status of contributed material

### ISSUE THREAD

**Issue by rbeyer** [DOI-USGS/ISIS3#588]
I can't find any statements about what copyright terms you accept material under.  Since ISIS is in the public domain, anything other than public domain contributions (with the exception of isolated 3rd party libraries, etc.) would complicate things.

Specifically, when I initiate a pull request, and you accept it, what's going on?  If I don't explicitly state that my contribution is in the public domain, it isn't (thanks Berne Convention Implementation Act of 1988).  That means that if a file or script was in the public domain (because it was written by USGS civil servants), and then I modified it (without attaching a copyright waiver statement), submitted a pull request, and you accepted it and merged it into your codebase, that file is now copyright me.

I think this should be addressed in two ways:

(1) There is currently no boilerplate public domain statement at the top of each file in the codebase, there needs to be.  Doesn't have to be long, but it needs to be there.

(2) Don't accept pull requests that don't have a copyright waiver.  You already have a history of having a comment block at the top of your files, which includes a description and then text blocks for each author.  In order to accept a pull request on a file, the submitter *MUST* add their name, contribution date, and description (just like your in-house authors do now) but should also add a copyright waiver that the Creative Commons CC0 public domain statement suggests, like so:

`Author:  Bob the Builder`
`Date:    2018-11-17`
`Description: Streamlined the program, improved documentation, etc.`
`To the extent possible under law, Bob the Builder has waived all copyright and related or `
` neighboring rights to their contribution to this file.  This work is published from`
`the United States.`

**Comment by jessemapel** [comment]
I think that #591 will resolve this.

```
In jurisdictions that recognize copyright laws, the author or authors
of this software dedicate any and all copyright interest in the
software to the public domain.
```

**Comment by jlaura** [comment]
I have put in a PR that adds PR templates to the repository.  Please see and comment on #596.

**Comment by jessemapel** [comment]
Fixed by #596
